modules/mcuboot/Kconfig

```diff
@@ -143,3 +143,13 @@ config MCUBOOT_USE_ALL_AVAILABLE_RAM
 	  RAM partition.
 
 endmenu
+
+config XIP_SPLIT_IMAGE
```

Comment:
Add a note in the readme that explains what the purpose of this extra config. I think it is important to explain that this config is not a global one and won't work outside the sample. I also think it is important to explain to the user that there is custom code (cmake, linker script, and partition manager configuration) in the sample.